validator/client/src/main/java/tech/pegasys/teku/validator/client/ValidatorClientService.java

```diff
@@ -271,14 +271,21 @@ public static ValidatorClientService create(
             })
         .exceptionally(
             error -> {
-              final Optional<Throwable> maybeCause =
-                  ExceptionUtil.getCause(error, InvalidConfigurationException.class);
-              if (maybeCause.isPresent()) {
-                LOG.warn(maybeCause.get().getMessage());
-              } else {
-                LOG.error("Error was encountered during validator client service start up.", error);
-              }
-              checkNoKeysLoaded(validatorConfig, validatorLoader);
+              ExceptionUtil.getCause(error, InvalidConfigurationException.class)
+                  .ifPresentOrElse(
+                      cause -> LOG.error(cause.getMessage()),
+                      () ->
+                          LOG.error(
+                              "Error was encountered during validator client service start up.",
+                              error));
+
+              LOG.error(
```

Comment:
I'm not sure about always logging this error, no matter where\why we throw the exception. So it could be misleading if it has nothing to do with keys.

Reply:
I can remove it, but this is the unhandled case we used to have so we'd have a stack trace... stack traces can be handy...

Reply:
I'll leave this as it is literally the old cause i've refactored to be cleaner..